clickhouse/tests/docker/docker-compose.yaml

```diff
@@ -29,155 +29,57 @@ services:
       - ./test_dictionary.xml:/etc/clickhouse-server/test_dictionary.xml
       - ./test.csv:/opt/dictionaries/test.csv
     depends_on:
-      - clickhouse-zookeeper
+      clickhouse-zookeeper:
+        condition: service_healthy
     networks:
       - clickhouse-network
 
   clickhouse-02:
-    image: ${CLICKHOUSE_REPOSITORY}:${CLICKHOUSE_VERSION}
+    << : *ch-node
     hostname: clickhouse-02
     container_name: clickhouse-02
-    cap_add:
-      - IPC_LOCK
-      - NET_ADMIN
-    ulimits:
-      nproc: 65536
-      nofile:
-        soft: 262144
-        hard: 262144
     ports:
       - "8129:8123"
       - "9002:9000"
-    volumes:
-      - ./override_configs/zookeeper.xml:/etc/clickhouse-server/config.d/zookeeper.xml
-      - ./override_configs/remote_servers.xml:/etc/clickhouse-server/config.d/remote_servers.xml
-      - ./override_configs/compression.xml:/etc/clickhouse-server/config.d/compression.xml
-      - ./metrika.xml:/etc/metrika.xml
-      - ./macros/macros-02.xml:/etc/clickhouse-server/config.d/macros.xml
```

Comment:
that's clever but I don't you can for the volumes since we are not mounting the same files for each service

- `./macros/macros-02.xml:/etc/clickhouse-server/config.d/macros.xml` for service 2
- `./macros/macros-03.xml:/etc/clickhouse-server/config.d/macros.xml` for service 3
- etc

Reply:
I think this is what was causing the failures for me locally, nice catch!

I've reintroduced the duplication for volumes, we really hit the limits of Yaml with this case.